python-template/{{cookiecutter.placeholder_repo_name}}/catalog-info.yaml

```diff
@@ -0,0 +1,32 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
```

Comment:
Non blocking, but do you have an ADR with catalog-info.yaml recommendations that you can link to?  Should we link to the data model for the catalog?

Reply:
Feanil and I are talking about the role of ADRs in OEPs.  I think we will find a way to publish OEP-55 so that the details of catalog-info.yaml are easily discoverable from this URL.